app/Locale/English.php

```diff
@@ -65,6 +65,11 @@ public function get_locale(): array
 			'FULLSCREEN_ENTER' => 'Enter Fullscreen',
 			'FULLSCREEN_EXIT' => 'Exit Fullscreen',
 
+			'SHARING_ALBUM_USERS' => 'Share this album with users',
+			'WAIT_FETCH_DATA' => 'Please wait while we get the data.',
+			'SHARING_ALBUM_USERS_NO_USERS' => "There's no user to share the album with.",
+			'SHARING_ALBUM_USERS_LONG_MESSAGE' => 'Select the users to share this album with them',
```

Comment:
A couple of comments about the language:
* Is there a reason why some of these strings end with a period (`.`) and some don't?
* Without checking how `WAIT_FETCH_DATA` is used, I would guess it should end with an ellipsis (`...`).
* `SHARING_ALBUM_USERS_NO_USERS` would probably look better in plural (`There are no users [...]`).
* Drop `them` from `SHARING_ALBUM_USERS_LONG_MESSAGE`.

Reply:
> Is there a reason why some of these strings end with a period (.) and some don't?

Not at all, it just looked better  at the time

Should be good now